### PR TITLE
Add test for organization mismatch should be project doesn't match (AST-79785)

### PR DESCRIPTION
**Pull request by elchananarb**
### Description

> dding a warning for the user when the SCM project doesn't match the CX project - TESTS

### References

>https://checkmarx.atlassian.net/browse/AST-79785

### Testing

> add ui test

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR (if applicable).
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [ ] The correct base branch is being used

### CODE REVIEW

File: src/main/java/com/checkmarx/intellij/Constants.java [hunk 1]
```diff
@@ -58,6 +58,8 @@ private Constants() {
     public static final String SELECTED_SCAN_PROPERTY = "Checkmarx.SelectedScan";
     public static final String RUNNING_SCAN_ID_PROPERTY = "Checkmarx.RunningScanId";
 
+    public static final String NOT_MATCH_PROJECT_NAME_DIFF_ORGANIZATION = "Phoenix/WebGoat";
```
Comment (AlvoBen) [inline src/main/java/com/checkmarx/intellij/Constants.java:61]: instead of this name, use random UUID

File: src/test/java/com/checkmarx/intellij/ui/TestTriggerScan.java [hunk 2]
```diff
@@ -68,6 +68,21 @@ public void testTriggerScanProjectAndBranchDontMatch() {
         Assertions.assertTrue(hasAnyComponent(BRANCH_DOES_NOT_MATCH));
     }
 
+    @Test
+    @Video
+    public void testTriggerScanProjectWithDifferentOrganizationsDontMatch() {
+
+        if (triggerScanNotAllowed()) return;
```
Comment (AlvoBen) [inline src/test/java/com/checkmarx/intellij/ui/TestTriggerScan.java:75]: Why return? this will make the test pass. 
you should fail the test. we should not get in a situation that create scan is not allowed in this test.

File: src/main/java/com/checkmarx/intellij/Constants.java (hunk 1 above)
Comment (elchananarb) [reply src/main/java/com/checkmarx/intellij/Constants.java:61]: The purpose of the test is to verify that if two projects have the same name but belong to different organizations, a warning is displayed. For example:

checkmarx/repoA
checkmarxDev/repoA

In our case, since the tests run on the project "pedrompflopes/WebGoat", I want to specifically check for "Phoenix/WebGoat", which is why I'm not using a UUID."

Comment (AlvoBen) [reply src/main/java/com/checkmarx/intellij/Constants.java:61]: so do: randomUUID = UUID.generateRandom.....
not matchProjectName == Strings.concat(randomUUID, "WebGoat").
also, This feild should not be declared in constants.java. it is being used only for your test. declared it locally in the test

Comment (AlvoBen) [reply src/main/java/com/checkmarx/intellij/Constants.java:61]: the purpose of the UUID is to be sure there will no be other project with this name. "Phoenix/WebGoat" is very specific and logical and there could be projects in the future with this name

File: src/test/java/com/checkmarx/intellij/ui/TestTriggerScan.java (hunk 2 above)
Comment (elchananarb) [reply src/test/java/com/checkmarx/intellij/ui/TestTriggerScan.java:75]: You are right, 
but this is how all the tests in this file. If we decide to change it, we should update all of them consistently. We need to understand why it was written this way before making any changes

Comment (AlvoBen) [reply src/test/java/com/checkmarx/intellij/ui/TestTriggerScan.java:75]: I do not think that we should continue with a bad practice when finding one.
But I do agree you should not change the rest of the test.
In my opinion you should make your test the right way and open a ticket to check the rest of the test. What do you think?

File: src/main/java/com/checkmarx/intellij/Constants.java (hunk 1 above)
Comment (elchananarb) [reply src/main/java/com/checkmarx/intellij/Constants.java:61]: We have no issue with having a project named Phoenix/WebGoat,
and the test will not fail even if such a name exists.
However, I will change it to an edge case where we modify the project name in the environment to Phoenix/WebGoat

File: src/test/java/com/checkmarx/intellij/ui/TestTriggerScan.java (hunk 2 above)
Comment (elchananarb) [reply src/test/java/com/checkmarx/intellij/ui/TestTriggerScan.java]: Okay, I'll start a task on this for the other testers to investigate.

**Review by AlvoBen — DISMISSED** [review]
